x-pack/plugins/observability_solution/infra/public/components/log_stream/log_stream_react_embeddable.tsx

```diff
@@ -101,6 +106,37 @@ export function getLogStreamEmbeddableFactory(services: Services) {
   return factory;
 }
 
+const DISMISSAL_STORAGE_KEY = 'observability:logStreamEmbeddableDeprecationCalloutDismissed';
+
+const DeprecationCallout = () => {
+  const [isDismissed, setDismissed] = useLocalStorage(DISMISSAL_STORAGE_KEY, false);
+
+  if (isDismissed) {
+    return null;
+  }
+
+  return (
+    <EuiCallOut
+      color="warning"
+      iconType="help"
+      onDismiss={() => setDismissed(true)}
+      css={{
+        position: 'absolute',
+        bottom: euiThemeVars.euiSizeM,
+        right: euiThemeVars.euiSizeM,
+        width: 'min(100%, 40ch)',
+      }}
+    >
+      <p>
+        {i18n.translate('xpack.infra.logsStreamEmbeddable.deprecationWarningDescription', {
+          defaultMessage:
+            'This panel is not longer actively maintained. You might prefer using saved searches for this visualization.',
```

Comment:
@mdbirnstiehl Could you double-check this callout description, please? 
It is used on logs stream embeddable, we want to ensure the user knows this panel won't be available for new dashboards but we won't discontinue its usage for existing dashboards. 
<img width="1251" alt="Screenshot 2024-10-01 at 10 33 06" src="https://github.com/user-attachments/assets/ab44d8eb-0003-4fbb-be6b-f291ae4a184c">

Reply:
How about:
Logs Stream panels are no longer maintained. Try using [saved searches](https://www.elastic.co/guide/en/kibana/current/save-open-search.html) for a similar visualization.